app/uk/gov/hmrc/trusts/controllers/CheckTrustController.scala

```diff
@@ -37,7 +38,7 @@ class CheckTrustController @Inject()(desService: DesService, config: AppConfig,
 
   def checkExistingTrust() = Action.async(parse.json) { implicit request =>
     import authService._
-    authorisedUser() {
+    authorisedUser() { userAffinityGroup: Option[AffinityGroup]=>
```

Comment:
Since we don't use this can we just do `_ =>`

Reply:
Done @adamconder